tests/bin/bats

```diff
@@ -57,6 +57,8 @@ export BATS_PREFIX="$(abs_dirname "$BATS_LIBEXEC")"
 export BATS_CWD="$(abs_dirname .)"
 export PATH="$BATS_LIBEXEC:$PATH"
 
+export OMP_NUM_THREADS=1
```

Comment:
I think this probably needs some kind of comment above it

Reply:
Done.

Reply:
Thanks